docs/manage/field-extractions/fer-templates/apache-access-logs.md

```diff
@@ -26,10 +26,10 @@ parse regex "^(?<src_ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})"
 
 | Field | Description | Example |
 |:--|:--|:--|
-| src_ip | IP address of the client (remote host) which made the request to the server | 221.125.19.252 |
-| method | Method used by the client | GET |
-| url | Resource requested by the client | `v` |
-| status_code | Status code that the server sends back to the client | 200 |
-| size | Size of the object returned to the client | 8825 |
-| referrer | Site that the client reports having been referred from | [http://www.google.com/url?sa=t&rct=j...source=web&cd=4](http://www.google.com/url?sa=t&rct=j&q=log-reduce&source=web&cd=4) |
-| user_agent  | Identifying information that the client browser reports about itself | Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10_6_7; en-us) AppleWebKit/533.21.1 (KHTML, like Gecko) Chrome/19.0.1084.30 Safari/536.5 |
+| src_ip | IP address of the client (remote host) from which the request is made to the server. | 221.125.19.252 |
+| method | Method used by the client. | GET |
+| url | Resource requested by the client. | `v` |
```

Comment:
should `v` be a url or is that intentional?

Reply:
Fixed the missing path.